klipper_config/menu.cfg

```
@@ -0,0 +1,38 @@
+### menu macros ###
+[menu __main __macros]
+type:list
+name: Macros
+
+[menu __main __macros __qgl]
+type: command
+enable: {not printer.idle_timeout.state == "Printing"}
+name: QGL
+gcode: quad_gantry_level
+
+[menu __main __macros __bedmesh]
+type: command
+enable: {not printer.idle_timeout.state == "Printing"}
+name: Bed Mesh
+gcode: bed_mesh
+
+[menu __main __macros __cleannozzle]
+type: command
+enable: {not printer.idle_timeout.state == "Printing"}
+name: Nozzle Scrub
+gcode: clean_nozzle
+
+[menu __main __macros __heatsoak]
+type: command
+enable: {not printer.idle_timeout.state == "Printing"}
+name: Heat Soak
+gcode: HEAT_SOAK
+
+[menu __main __macros __caselightson]
+type: command
+name: Case Lights On
+gcode: caselights_on
+
+[menu __main __macros __caselightsoff]
+type: command
+name: Case Lights Off
+gcode: caselights_off
```

Add a new line at the end of the file please
```suggestion
gcode: caselights_off

```